// gitmoji-cli/lib/commands/config/prompts.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = exports.EMOJI_COMMIT_FORMATS = exports.CONFIGURATION_PROMPT_NAMES = void 0;
const CONFIGURATION_PROMPT_NAMES = {
  AUTO_ADD: 'autoAdd',
  EMOJI_FORMAT: 'emojiFormat',
  SCOPE_PROMPT: 'scopePrompt',
  SIGNED_COMMIT: 'signedCommit'
};
exports.CONFIGURATION_PROMPT_NAMES = CONFIGURATION_PROMPT_NAMES;
const EMOJI_COMMIT_FORMATS = {
  CODE: 'code',
  EMOJI: 'emoji'
};
exports.EMOJI_COMMIT_FORMATS = EMOJI_COMMIT_FORMATS;
var _default = [{
  name: CONFIGURATION_PROMPT_NAMES.AUTO_ADD,
  message: 'Enable automatic "git add ."',
  type: 'confirm'
}, {
  name: CONFIGURATION_PROMPT_NAMES.EMOJI_FORMAT,
  message: 'Select how emojis should be used in commits',
  type: 'list',
  choices: [{
    name: ':smile:',
    value: EMOJI_COMMIT_FORMATS.CODE
  }, {
    name: '😄',
    value: EMOJI_COMMIT_FORMATS.EMOJI
  }]
}, {
  name: CONFIGURATION_PROMPT_NAMES.SIGNED_COMMIT,
  message: 'Enable signed commits',
  type: 'confirm'
}, {
  name: CONFIGURATION_PROMPT_NAMES.SCOPE_PROMPT,
  message: 'Enable scope prompt',
  type: 'confirm'
}];
exports.default = _default;
